include/boost/geometry/iterators/point_iterator.hpp

```diff
@@ -245,33 +246,26 @@ struct points_end<MultiPolygon, multi_polygon_tag>
 // MK:: need to add doc here
 template <typename Geometry>
 class point_iterator
-    : public detail::point_iterator::iterator_type<Geometry>::type
+    : public boost::iterator_adaptor <
+        point_iterator<Geometry>
+      , typename detail::point_iterator::iterator_type<Geometry>::type
+    >
 {
 private:
     typedef typename detail::point_iterator::iterator_type<Geometry>::type base;
```

Comment:
The type `base` is not really needed now that you are using `boost::iterator_adaptor<>`. It is defined in `boost::iterator_adaptor<>` as `base_type`.